src/datafile.rkt

```diff
@@ -206,6 +204,14 @@
       (error 'merge-datafiles "Cannot merge datafiles at different ~a" f)))
   (unless dirs
     (set! dirs (map (const #f) dfs)))
+  (define tests
+    (for/list ([df (in-list dfs)] [dir (in-list dirs)]
+                                  #:when true
+                                  [test (in-list (report-info-tests df))])
```

Comment:
Here the `#:when` and the next line should both be aligned with the `[` in `[df`.